ui/src/components/WebRTCVideo.tsx

```diff
@@ -724,15 +724,15 @@ export default function WebRTCVideo() {
                                 hdmiError ||
                                 peerConnectionState !== "connected",
                               "!opacity-60": showPointerLockBar,
-                              "animate-slideUpFade border border-slate-800/30 opacity-0 shadow dark:border-slate-300/20":
```

Comment:
Note that we no longer _can_ or _should_ declare the `opacity-0` in the source, as the `slideUpFade` implies and enforces it. If you leave it in here, the contents of the all the little dialog boxes never appears when clicking.

This change happens a lot of places, and _is_ required.